app/main.py

```diff
@@ -17,7 +20,12 @@ async def lifespan(app: FastAPI):
     yield
 
 
-app = FastAPI(default_response_class=ORJSONResponse, lifespan=lifespan)
+app = FastAPI(
+    default_response_class=ORJSONResponse,
+    lifespan=lifespan,
+    docs_url=None,
+    redoc_url=None,
```

Comment:
we don't want docs?

Reply:
aha, maybe add a comment here that we'll override the docs with our own special doc (i.e. the spec json)